modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -0,0 +1,251 @@
+However, it can still be used on APOC 2025.03 if queries are prepended with `CYPHER 5`, or if the database’s default version is set to `CYPHER 5`.
+In this case, Neo4j will use APOC and Cypher as they existed at the time of the 5.26 long-term support (LTS) release.
+
+.Using a procedure removed in Cypher 25 with APOC 2025.03+
+[source, cypher]
+----
+CYPHER 5
+CALL apoc.create.uuids(10)
+----
+
+[[cypher-selection-with-other-query-options]]
+=== Combine Cypher version selection with other query options
+
+It is possible to combine Cypher version selection with other xref:planning-and-tuning/query-tuning.adoc[query options].
+The below example selects both the version and the xref:planning-and-tuning/runtimes/concepts.adoc[runtime] of Cypher for the same query:
+
+.Combining Cypher version selection with other query options
+[source, cypher]
+----
+CYPHER 5 runtime=parallel
+MATCH (n:Person)
+RETURN n.name
+----
+
+[[select-default-cypher-version]]
+== Select the default Cypher version when creating a database
+
+Databases created on Neo4j 2025.03 or later will have Cypher 25 as their default language.
+This is true for the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/#manage-database-systems[system, standard, and composite] Neo4j databases.
+
+To select a different default Cypher version for a database, add `DEFAULT LANGUAGE <language version>` to the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/create-databases/[`CREATE DATABASE`] statement.
+This can be done on any Neo4j database.
+
+[NOTE]
+Selecting a default Cypher version on a database requires the `SET DEFAULT LANGUAGE` privilege.
+
+Selecting `CYPHER 5` as the default database language ensures that every query run on that database uses the language as it existed at the time of the Neo4j 5.26 long-term support release.
+Any changes introduced in Neo4j 2025.03 or later will not affect the semantics of the query.
+
+.Create a database with Cypher 5 as the default language
+[source, cypher]
+----
+CREATE DATABASE my_database DEFAULT LANGUAGE CYPHER 5
+----
+
+Selecting `CYPHER 25` ensures that the query will be executed using the language as it exists in the version of Neo4j that the database is currently running, provided it is on Neo4j 2025.03 or later.
+
+.Create a database with Cypher 25 as the default language
+[source, cypher]
+----
+CREATE DATABASE my_new_database DEFAULT LANGUAGE CYPHER 25
+----
+
+[[alter-default-cypher-version]]
+== Alter the default Cypher version for a database
+
+To alter the default Cypher version on a database, add `SET DEFAULT LANGUAGE <language version>` to the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/alter-databases/[`ALTER DATABASE`] command.
+
+[NOTE]
+Selecting a default Cypher version on a database requires the `SET DEFAULT LANGUAGE` privilege.
```

Comment:
we renamed the privilege
```suggestion
Selecting a default Cypher version on a database requires the `SET DATABASE DEFAULT LANGUAGE` privilege.
```
so that if we want to introduce a privilege for the remote alias case there isn't some confusion around which privilege steers what